portfolio/src/main/webapp/script.js

```diff
@@ -16,6 +16,7 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
```

Comment:
You can probably remove this method since its no longer being used?

Reply:
Ok, I removed the method in latest push